src/AppBundle/Entity/City.php

```diff
@@ -40,6 +40,13 @@ class City
      */
     protected $connections;
 
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="sender_email", type="string", nullable=true)
```

Comment:
But this is not an optional naming. If you leave it the column name will be exactly that.
No. It's not a big deal but since you should set the field as not nullable you can alter this too.